icu4c/source/data/unidata/create_build_headers.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# © 2021 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+# This script runs Bazel to create (generate) header files from data that are
+# needed for bootstrapping the ICU4C build to integrate the data.
+
+# Required environment variables:
+#   - ICU_SRC - the root directory of ICU source. This directory contains the
+#               `icu4c` directory.
+
+
+bazelisk build //icu4c/source/common:normalizer2
+cp $ICU_SRC/bazel-bin/icu4c/source/common/norm2_nfc_data.h $ICU_SRC/icu4c/source/common/norm2_nfc_data.h
+
+bazelisk run //tools/unicode/c/genprops $ICU_SRC/icu4c/
```

Comment:
Please add commands to copy the outputs from genprops back into $ICU_SRC.

Reply:
Actually, for now, genprops takes one folder for both input and output. It will write into the source tree, despite what Bazel likes. Might work for now, at least until we start also building and invoking genuca. Eventually genprops might need another parameter for the output folder.